### PR TITLE
Search updates nov 2022

### PR DESCRIPTION
**Pull request by oosacker**

### CODE REVIEW

File: src/Job/SearchIndexJob.php
Comment (oosacker) [inline src/Job/SearchIndexJob.php]: probably dont need this actually

**Review by chtombleson — APPROVED** [review]
We should also tag a new version :D once merged

**Comment by oosacker** [comment]
> We should also tag a new version :D once merged

I am waiting for reply from HPA before I merge (it's on Alcohol Resource UAT now), also I am not sure how it will work on other sites, will need to test well when updating the module

**Pull request merged**
